sigmf-spec.md

```diff
@@ -200,10 +200,15 @@ Rules for SigMF Collection files:
 Rules for SigMF Archive files:
 1. The Archive MUST use the `tar` archive format, as specified by POSIX.1-2001.
 1. The Archive file's filename extension MUST be `.sigmf`.
-1. The Archive MUST contain the following files: for each contained Recording
-   with some name given here meta-syntactically as `N`, files named `N` (a
-   directory), `N/N.sigmf-meta`, and `N/N.sigmf-data`.
-1. The Archive MAY contain a `.sigmf-collection` file in the top-level directory.
+1. The Archive MUST contain at least one SigMF Recording.
+1. The Archive MAY organize SigMF Recordings into subdirectories such that for
+   each contained Recording with some base name `N`, individual files are stored
+   within the directory `N` (e.g.: `N/N.sigmf-meta`, and `N/N.sigmf-data`). The
+   use of subdirectories within an archive is NOT RECOMMENDED and may be
+   deprecated in future versions of SigMF.
+1. The Archive MAY contain a `.sigmf-collection` file in the top-level directory,
```

Comment:
I like the old version of this rule.

Reply:
This was updated to ensure visibility on the fact that sigmf-collection files MUST NOT exist with the subdirectory permission above.